ouroboros-consensus/src/Ouroboros/Consensus/Protocol/Abstract.hs

```diff
@@ -122,7 +122,7 @@ class ( Show (ChainState    p)
   type family ValidationErr p :: *
 
   -- | Blocks that the protocol can run on
-  type family SupportedBlock p :: * -> Constraint
+  type family SupportedHeader p :: * -> Constraint
```

Comment:
Is this only meant for headers or do we also use it with blocks sometimes?

Reply:
Perhaps we might apply it to blocks in tests (like in the model of the chain DB where we don't distinguish).